docs/sdks_03_safe_apps.md

```diff
@@ -29,6 +44,22 @@ We provide Safe App developers with reusable react components to make it easy to
 [UI Guidelines](assets/safe_apps_guides.png)
 
 
+### Existing Safe apps
```

Comment:
We need to agree on capitalized A or not :D cc @lukasschor

Reply:
capital A sounds good, will change it.